web/src/containers/admin/StateAdmin.test.js

```diff
@@ -95,52 +140,65 @@ describe('<StateAdmin />', () => {
   describe('with a pending request', () => {
     beforeEach(() => {
       const props = {
-        currentState: {
-          id: 'md',
-          name: 'Maryland'
-        },
-        affiliations: [requestedAffiliation],
-        roleTypes: [{ id: 50, name: 'eAPD Federal Admin' }],
         getRoleTypes: jest.fn(),
         getStateAffiliations: jest.fn(),
         updateStateAffiliation: jest.fn()
       };
-      renderUtils = renderWithConnection(<StateAdmin {...props} />);
+      const stateId = initialState.user.data.state.id;
+      fetchMock
+        .onGet(`/states/${stateId}/affiliations?status=pending`)
+        .reply(200, [requestedAffiliation]);
+      fetchMock
+        .onPatch(`/states/${stateId}/affiliations/${requestedAffiliation.id}`)
+        .reply(200);
+      renderUtils = renderWithConnection(<StateAdmin {...props} />, {
+        initialState
+      });
     });
 
-    test('renders name, email, phone', () => {
+    it('renders name, email, phone', async () => {
       // Note: we render the affiliations in each tab and update on tab change, so we
       // expect to see 3 instances which is why this is using getAllByText
       const { getAllByText } = renderUtils;
-      expect(getAllByText(requestedAffiliation.displayName)).toHaveLength(3);
+      await waitFor(() => {
+        expect(getAllByText(requestedAffiliation.displayName)).toHaveLength(3);
+      });
       expect(getAllByText(requestedAffiliation.email)).toHaveLength(3);
       expect(getAllByText(requestedAffiliation.primaryPhone)).toHaveLength(3);
     });
 
-    test('renders approve and deny buttons', () => {
+    it('renders approve and deny buttons', async () => {
       const { getByText } = renderUtils;
-      expect(getByText('Approve')).toBeTruthy();
+      await waitFor(() => {
+        expect(getByText('Approve')).toBeTruthy();
+      });
       expect(getByText('Deny')).toBeTruthy();
     });
 
     it('should open manage modal on approve', async () => {
       const { getByText } = renderUtils;
+      await waitFor(() => {
+        expect(getByText('Approve')).toBeTruthy();
+      });
       fireEvent.click(getByText('Approve'));
       await waitFor(() => {
         expect(getByText('Edit Permissions')).toBeInTheDocument();
       });
     });
 
-    it('should open confirmation modal on deny', async () => {
+    xit('should open confirmation modal on deny', async () => {
```

Comment:
should these just be deleted?  Why keep them?  Will they be turned on in the future?